documents/attribute.go

```diff
@@ -16,6 +17,42 @@ func (a attributeType) String() string {
 	return string(a)
 }
 
+// AttrKey represents a sha256 hash of a attribute label given by a user.
+type AttrKey [32]byte
```

Comment:
Why is it public?

Is it not more an internal representation in coredocument.

Outside of the package the key should be seen as a normal string